tests/system_tests/backend_fixtures.py

```diff
@@ -163,6 +196,37 @@ def make_org(self, name: str | None = None, *, username: str) -> str:
         )
         return name
 
+    def make_team(
+        self,
+        name: str | None = None,
+        *,
+        username: str,
+        org_name: str | None = None,
+        plan_name: str | None = None,
+    ) -> TeamAndOrgNames:
+        """Create a new team and return the team name."""
+        name = name or f"team-{self.worker_id}-{random_string()}"
+        org_name = org_name or f"org-{self.worker_id}-{random_string()}"
+        plan_name = plan_name or f"plan-{self.worker_id}-{random_string()}"
+
+        self.send_cmds(
+            TeamCmd(
+                "up",
+                username=username,
+                fixtureData=TeamOrgState(
+                    planName=plan_name,
+                    team=_Team(name=name),
+                    organization=_Organization(name=org_name),
+                ),
+            ),
+        )
+
+        # Register command(s) to delete the team(s) on cleanup
+        self._cleanup_stack.append(
+            TeamCmd("down"),
+        )
```

Comment:
nit: this is short enough to fit on one line and be readable


```suggestion
        self._cleanup_stack.append(TeamCmd("down"))
```